src/types/claim.ts

```diff
@@ -1,13 +1,15 @@
-import type { PropertyId } from './entity.js'
+import type { Guid, PropertyId } from './entity.js'
 import type { SnakValue } from './snakvalue.js'
-import type { parsers } from '../helpers/parse_claim.js'
+import type { parsers } from '../helpers/parse_snak.js'
 
 export type Rank = 'normal' | 'preferred' | 'deprecated'
 export type SnakType = 'value' | 'somevalue' | 'novalue'
-export type DataType = keyof typeof parsers
+
+export type InconsistentDataType = 'globecoordinate' | 'musical notation'
+export type DataType = keyof typeof parsers | InconsistentDataType
```

Comment:
The current way of `keyof typeof parsers` is already horrible and there should be an explicit list (like #106 uses). That's not the thing that this PR should take care of but we should keep it in mind for the future.

Personally I think the InconsitentDataType thingies should be ignored type errors as they were actually bugs in the Wikibase / Wikidata API we have to deal with and not useful DataTypes. `// @ts-expect-error` seems more appropriate to me in that case

Reply:
done in 0056c1b